STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
+if the update is 'nothing new') as a way to ensure they are active
+and have the support needed.
+
```

Comment:
what resources do initiatives have? i see some initiatives that have their own repo, and some that currently exist only in an issue here. might be useful to plan out a formal section on what infrastructure an initiative gets

Reply:
100% agreed - probably something we should do once this lands?

I know that NodeTogether has some financial and other support directly from the Foundation - @rachelnicole may be able to shed more light on that.